recipes/python-libsbml-experimental/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "python-libsbml-experimental" %}
```

Comment:
Why is this called experimental when the package is just `python-libsbml`?

Reply:
the archive contains sources for both packages, an environment variable controls which package is to be built.

Reply:
Can you please make a note of that in a comment

Reply:
done